From d19e00ae49f654b90f26f4c7828f74ea1b664343 Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Thu, 24 Oct 2013 08:13:34 +0200
Subject: [PATCH] lxc-alpine: run bootmisc and syslog at boot runlevel

The bootmisc script is needed to clean up various temp dirs like /tmp
and migrate /var/run to /run if needed.

The syslog service is started in 'boot' runlevel when running on real
hardware so we do the same for containers.

Signed-off-by: Natanael Copa <ncopa@alpinelinux.org>
---
 templates/lxc-alpine.in | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/templates/lxc-alpine.in b/templates/lxc-alpine.in
index 8600a34..2756b89 100644
--- a/templates/lxc-alpine.in
+++ b/templates/lxc-alpine.in
@@ -140,7 +140,8 @@ EOF
     mknod -m 666 "$rootfs/dev/ptmx" c 5 2
 
     # start services
-    ln -s /etc/init.d/syslog "$rootfs"/etc/runlevels/default/syslog
+    ln -s /etc/init.d/bootmisc "$rootfs"/etc/runlevels/boot/bootmisc
+    ln -s /etc/init.d/syslog "$rootfs"/etc/runlevels/boot/syslog
 
     return 0
 }
-- 
1.8.4.1

